toolkit/geo-compose/src/main/java/com/arcgismaps/toolkit/geocompose/GeoViewProxy.kt

```diff
@@ -17,10 +17,218 @@
+     * be initialized by the composable component when it enters the composition and set to null when
+     * it is disposed by calling [setGeoView].
+     *
+     * @since 200.3.0
+     */
+    private var geoView: GeoView? = null
+
+    /**
+     * Sets the [geoView] parameter on this operator. This should be called by the composable component
+     * when it enters the composition and set to null when it is disposed.
+     *
+     * @since 200.3.0
+     */
+    protected fun setGeoView(geoView: GeoView?) {
+        this.geoView = geoView
+    }
+
+    private val nullGeoViewErrorMessage: String =
+        "$classNameForErrorMessage must be part of the composition when this member is called."
+
+
+    /**
+     * Initiate an Identify operation on the specified [graphicsOverlay].
+     *
+     * The [tolerance] parameter determines the extent of the region used during the identify operation. Any graphics
+     * that intersect this test region when rendered are returned, up to the [maximumResults] limit. A tolerance
+     * of 0 tests just the physical pixel at [screenCoordinate].
+     * Tolerance values above 0 are in [Dp] and specify a circular region centered on [screenCoordinate], with radius equal
+     * to [tolerance]. The maximum allowed tolerance value is 100 [Dp], resulting in an identify circle of diameter
+     * 200 [Dp].
+     *
+     * The [returnPopupsOnly] parameter controls what properties are populated in the [IdentifyGraphicsOverlayResult]
+     * instance that is returned by the identify operation:
+     * * true: only the [IdentifyGraphicsOverlayResult.popups] property will be populated with results. If the overlay
+     * does not have popups an error will be returned.
+     * * false: the [IdentifyGraphicsOverlayResult.graphics] property will be populated, and the
+     * [IdentifyGraphicsOverlayResult.popups] property will be populated if the overlay has popups.
+     *
+     * @param graphicsOverlay overlay on which to run the identify
+     * @param screenCoordinate location at which to run identify in screen coordinates
+     * @param tolerance extent of the region used during the identify operation
+     * @param returnPopupsOnly whether the graphics property of the result is populated
+     * @param maximumResults maximum size of the result set of graphics to return. A null value indicates unlimited results
+     * @return A [Result] containing an [IdentifyGraphicsOverlayResult], or failure
+     * @throws IllegalArgumentException if [maximumResults] is less than zero
+     * @since 200.4.0
+     */
+    public suspend fun identify(
+        graphicsOverlay: GraphicsOverlay,
+        screenCoordinate: ScreenCoordinate,
+        tolerance: Dp,
+        returnPopupsOnly: Boolean = false,
+        maximumResults: Int? = 1
+    ): Result<IdentifyGraphicsOverlayResult> {
+        val maximumResultsSanitized = if (maximumResults == null || maximumResults < 0) {
```

Comment:
I think it should be:
```suggestion
        val maximumResultsSanitized = if (maximumResults == null || maximumResults <= 0) {
```